ballerina/vector_store.bal

```diff
@@ -83,48 +89,65 @@ public isolated class VectorStore {
 
     # Deletes vector entries from the store by their reference document ID.
     #
-    # + id - The ID of the vector entry to delete.
-    # + return - An `ai:Error` if the deletion fails; otherwise, `()` is returned indicating success.
-    public isolated function delete(string id) returns ai:Error? {
+    # + ids - The ID/s of the vector entries to delete
```

Comment:
```suggestion
    # + ids - One or more identifiers of the vector entries to delete
```

Reply:
Fixed in [ec3d53a](https://github.com/ballerina-platform/module-ballerinax-ai.milvus/pull/8/commits/ec3d53a6469ab718121bf1f7e57350be2219aa81)